tasks/eth/2-add-superchainConfig/Validation.md

```diff
@@ -0,0 +1,108 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction.
+
+Please ensure that the following changes (and none others) are made to each contract in the system.
+The "Before" values are excluded from this list and need not be validated.
+
+## Background
+
+The `SuperchainConfigProxy` has already been deployed and initialized. It's address
+`0x95703e0982140d16f8eba6d158fccede42f04a4c` appears in the state diff below.
+
+## State Overrides
+
+There should also be a single 'State Override' in the Foundation Safe contract
+(`0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A`) to enable the simulation by reducing the threshold to
+1:
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000004` <br/>
+  **Value:** `0x0000000000000000000000000000000000000000000000000000000000000001`
+
+## State Changes
+
+##### `0x229047fed2591dbec1ef1118d64f7af3db9eb290` (`SystemConfigProxy`)
+
+- **Key:** `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc` <br/>
+  **Value:** `0x00000000000000000000000033a032ec93ec0c492ec4bf0b30d5f51986e5a314` <br/>
+  **Meaning:** Implementation address is set to the new `SystemConfig`.
```

Comment:
This will be addresses by providing instructions for using diffyscan to validate the diff between the repo source and verified etherscan source.
That is WIP, so resolving this (and similar comments) to declutter.